lib/imageResize.ts

```diff
@@ -13,6 +21,12 @@ export const ImageResize = Image.extend({
             : `${element.style.cssText}`;
         },
       },
+      wrapperStyle: {
+        default: `display: ${inline ? "inline-block" : "flex"};`, // Default style for the wrapper
```

Comment:
```suggestion
        default: `${ inline ? 'display: inline-block; float: left; padding-right: 8px;' : 'display: flex'}`
```
I think the default style code should match the style applied when clicking the left-align button.
Otherwise, it has minor errors when writing text.